torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -303,7 +315,7 @@ def _(ctx, subgraph, identifier, operands):
 @invoke_subgraph.py_impl(FakeTensorMode)
 def _(mode, subgraph, identifier, operands):
     # TODO(anijain2305) - Implement fake tensor caching.
-    with mode:
+    with mode, mode.shape_env.ignore_fresh_unbacked_symbols():
```

Comment:
I don't know much about pending unbacked, @bobrenjc93 can you confirm that this is just a feature to prevent people from writing incorrect metas?

Reply:
IIUC this is less about if the symbols occur in the output but rather it's about ensuring that the binding sites are encoded in the fx graph. I'm not too familiar with this mark_compiled_region work. Is this implemented as a HOP? If so this is a common reason to use ignore_fresh_unbacked_symbols. You can see @ydwu4 do something similar for while loops in https://github.com/pytorch/pytorch/pull/142031/files.